launcher/ui/dialogs/OfflineLoginDialog.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2013-2021 MultiMC Contributors
```

Comment:
ditto

Reply:
License header removed in https://github.com/PolyMC/PolyMC/pull/50/commits/6ecc8c5496cd1fa121b69f770c0664320fd7dc1d.